apply/default/routes.py

```diff
@@ -44,6 +44,7 @@ def index_fund_round(fund_short_name, round_short_name):
         migration_banner_enabled=Config.MIGRATION_BANNER_ENABLED,
         is_expression_of_interest=round_data.is_expression_of_interest,
         link_to_contact_us_page=round_data.reference_contact_page_over_email,
+        support_desk_apply=Config.SUPPORT_DESK_APPLY,
```

Comment:
What you could have done is add `SUPPORT_DESK_APPLY` and `SUPPORT_DESK_ASSESS` to the "globla" template context, so you always have them available in the templates without specifically providing it each time.

Reply:
Very good shout, thanks @albertkol ❤️, did the changes.